Archive Auction and Crowdloans (#6261)
w3f/polkadot-wiki: docs/general/polkadot-v1.md

### Polkadot's Additional Functionalities

> The Polkadot relay-chain also manges [crowdloans](../learn/archive/learn-crowdloans.md),

Reviewer: ```suggestion
The Polkadot relay-chain managed [crowdloans](../learn/archive/learn-crowdloans.md),
```

[auctions](../learn/archive/learn-auction.md), [staking](../learn/learn-staking.md),
[accounts](./learn-accounts-index), [balances](../learn/learn-transactions.md#balance-transfers),

> and [governance](../learn/learn-polkadot-opengov.md). Relay chain cores are leased in 6-month chunks

Reviewer: ```suggestion
and [governance](../learn/learn-polkadot-opengov.md). Relay chain cores were leased in 3-month chunks
```

> for a maximum of two years, and crowdloans allow users to trustlessly loan funds to teams for lease

Reviewer: ```suggestion
for a maximum of two years, and crowdloans allowed users to trustlessly loan funds to teams for lease
```

> deposits in exchange for pre-sale tokens. There is no other way you could use Polkadot 1.0.

Reviewer: ```suggestion
deposits in exchange for pre-sale tokens. There was no other way you could access cores on Polkadot 1.0.
```

balances transfers, staking, and governance, can be delegated to
[system parachains](../learn/learn-system-chains.md) to increase flexibility and to focus the use of
the relay-chain to provide shared security and interoperability. Blockspace is only accessible

> through auctions, but an auction winner has access to a "freighter of blocks" regardless it is

Reviewer: Looking at the wording in this doc on Polkadot v1, it makes sense to talk about auctions and crowdloans in past tense.

Author: will go through that page in more detail and check
